src/snappy_device_agents/devices/multi/multi.py

```diff
@@ -78,6 +81,18 @@ def provision(self):
 
         self.save_job_list_file()
 
+    def this_job_complete(self):
```

Comment:
It's a little contrived, but still useful I think. I added a test for this. Most device agents don't have unit tests historically, since they often have to interact with hardware, reboot mechanisms, etc. But I'm trying to look for ways to abstract them enough to make them more testable.